packages/hunspell.rb

```diff
@@ -12,27 +13,24 @@ class Hunspell < Package
   is_fake
 
   if ARGV[0] == 'install'
-    puts
-    puts 'Enter your preferred language:'
-    puts '1 = American English'
-    puts '2 = Français'
-    puts '3 = Español'
-    puts '0 = Cancel'
-
-    while version = $stdin.gets.chomp
-      case version
-      when '1'
-        depends_on 'hunspell_en_us'
-        break
-      when '2'
-        depends_on 'hunspell_fr_fr'
-        break
-      when '3'
-        depends_on 'hunspell_es_any'
-        break
-      when '0'
-        abort
+    begin
+      @timeout = 10
+      @user_input = Timeout.timeout(@timeout) do
+        puts
+        puts 'Enter your preferred language:'
+        puts "(The default, American English, will be selected in #{@timeout} seconds if there is no input.)"
+        puts '1 = American English'
+        puts '2 = Français'
+        puts '3 = Español'
+        puts '0 = Cancel'
+        $stdin.gets.chomp
       end
+    rescue Timeout::Error
```

Comment:
How about combining the `do...end` block and `rescue` block?
```ruby
@user_input = Timeout.timeout(10) do
  ...
rescue
  ...
end
```

Reply:
Can you verify if that works?

Reply:
What happens if an incorrect value is entered?

Reply:
With @supechicken's changes, an incorrect value uses the default.